modules/admin_manual/pages/enterprise/reporting/metrics.adoc

```diff
@@ -30,6 +30,8 @@ special setup in order to not put the production database under huge load when g
 this, please replicate your installation (application server + read-only database) and install/use the
 Metrics App on the replica.
 
+NOTE: Consider that IE 11 is not compatible with Metrics, because new web technologies have been used that are not supported by IE 11.
```

Comment:
```suggestion
NOTE: IE 11 is not compatible with Metrics, because new web technologies have been used that are not supported by IE 11.
```
IMO less words reads better.

Reply:
Agreed.